index.html

```diff
@@ -280,28 +222,63 @@ <h4 class="frame-name">CSS</h4>
             <li class="tabs">
               <h4 class="tab-title">Framework</h4>
               <a href="#">
-
                 <img src="./image/right.svg" alt="right" class="right-arrow" />
-
-                <img src="./image/right.png" alt="right" class="right-arrow" />
-
               </a>
             </li>
           </ul>
           <ul class="pro-languages separator-bottom">
             <li class="tabs">
               <h4 class="tab-title">Skills</h4>
               <a href="#">
-
                 <img src="./image/right.svg" alt="right" class="right-arrow" />
-
-                <img src="./image/right.png" alt="right" class="right-arrow" />
-
               </a>
             </li>
           </ul>
         </div>
       </div>
     </section>
+    <section class="form">
+      <ul class="form-header">
+        <li class="form-list">
+          <h4 class="contact-title">Contact me</h4>
+          <p class="contact-description">
+            If you have an application you are interested in developing, a
+            feature that you need built or a project that needs coding. I’d love
+            to help with it.
+          </p>
```

Comment:
As the previous reviewer requested, please align the text into the middle you can use `text-align:center`to do this.



<img width="395" alt="Screenshot 2023-01-17 at 9 52 22 PM" src="https://user-images.githubusercontent.com/25887253/212987758-c7b50626-1dfd-4dde-8ffd-5bcb16bcafaf.png">